ehitchhiking-ui/src/app/pages/main-screen/Route.ts

```diff
@@ -8,4 +10,5 @@ interface Route {
   routeColor: string;
   passengers: any[];
   driverRating: number;
+  carsSelect: Car
```

Comment:
bad naming: carsSelect. Pls rename to car

Reply:
fixed